### PR TITLE
Update beta version and add node-less

### PR DESCRIPTION
**Pull request by PierreSchwang**
Hi there!

A few hours ago v7.0_beta3 was released which seems to fix some issues regarding to LDAP / AD.
Also, I added node-less to support the internal stylesheet editor (change colors, whatever)

### CODE REVIEW

File: 7-beta/Dockerfile [hunk 1]
```diff
@@ -54,6 +54,8 @@ RUN curl -o phantomjs.tar.bz2 -SL https://bitbucket.org/ariya/phantomjs/download
 RUN curl -sS https://getcomposer.org/installer \
     | php -- --version=1.10.17 --install-dir=/usr/local/bin --filename=composer
 
+RUN apt update ; apt install -qy node-less ; rm -rf /var/lib/apt/* /var/cache/apt/*
```
Comment (corro) [inline 7-beta/Dockerfile]: Please add `node-less` to the list of apt dependencies on line 5 instead.

**Review by corro — CHANGES_REQUESTED** [review]
Hi @PierreSchwang

Thank you for your suggestions! I will happily merge this PR when you address the minor objection I have (see review).

File: 7-beta/Dockerfile (hunk 1 above)
Comment (PierreSchwang) [reply 7-beta/Dockerfile]: Sure, thank you. I must have missed that somehow.
I also added, that the apt cache will be removed as well

**Comment by corro** [comment]
@PierreSchwang thank you for your contribution!

**Pull request merged**